src/carbonio-files-ui-common/utils/utils.ts

```diff
@@ -101,11 +101,11 @@ export const getIconByFileType = (type: NodeType, subType?: Maybe<string>): stri
 		case NodeType.Root: {
 			switch (subType) {
 				case ROOTS.LOCAL_ROOT:
-					return 'Home';
+					return 'FolderOutline';
 				case ROOTS.TRASH:
-					return 'Trash2';
+					return 'Trash2Outline';
 				case ROOTS.SHARED_WITH_ME:
-					return 'Share';
+					return 'ArrowCircleLeftOutline';
```

Comment:
Not sure if this icon should change, it wasn't requested in the issue. Point it out to @eugeniamuff  and wait fot her validation